### ISSUE TITLE
Option `Use custom` does not work

### ISSUE THREAD

**Issue by Samet-MohamedAmin** [flathub/org.raspberrypi.rpi-imager#11]
I want to choose custom image.

So I followed these steps:
1. I clicked on `CHOOSE OS` 
2. It opens `Operating System` options
3. I choose the option `Use Custom`
4. It opens a window to choose custom image file
5. I selected an image file with `.img` extension
6. It returns to `Operating System` options without doing anything

The end result looks like this

![](https://i.imgur.com/OLqw7Ur.png)

---
**UDATE**
system and app info
```
[ ]$ cat /etc/*release
Fedora release 34 (Thirty Four)
NAME=Fedora
VERSION="34.20210320.n.1 (Silverblue Prerelease)"
ID=fedora
VERSION_ID=34
VERSION_CODENAME=""
PLATFORM_ID="platform:f34"
PRETTY_NAME="Fedora 34.20210320.n.1 (Silverblue Prerelease)"
ANSI_COLOR="0;38;2;60;110;180"
LOGO=fedora-logo-icon
CPE_NAME="cpe:/o:fedoraproject:fedora:34"
HOME_URL="https://fedoraproject.org/"
DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora-silverblue/"
SUPPORT_URL="https://fedoraproject.org/wiki/Communicating_and_getting_help"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=34
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=34
PRIVACY_POLICY_URL="https://fedoraproject.org/wiki/Legal:PrivacyPolicy"
VARIANT="Silverblue"
VARIANT_ID=silverblue
OSTREE_VERSION='34.20210320.n.1'
Fedora release 34 (Thirty Four)
Fedora release 34 (Thirty Four)
```
```
[ ]$ flatpak --version
Flatpak 1.10.2
```
```
[ ]$ flatpak info org.raspberrypi.rpi-imager 

Raspberry Pi Imager - Raspberry Pi imaging utility

          ID: org.raspberrypi.rpi-imager
         Ref: app/org.raspberrypi.rpi-imager/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 1.6
     License: Apache-2.0
      Origin: flathub
  Collection: org.flathub.Stable
Installation: system
   Installed: 1.0 MB
     Runtime: org.kde.Platform/x86_64/5.15
         Sdk: org.kde.Sdk/x86_64/5.15

      Commit: e33f4d26a6fffdc2d6c2ff7dfe2e66faec4d7e3f4f5b2d91b84b4d3b75ca134e
      Parent: 903ac26739883beff9a797a4fb98721dacbacc0a2bbff27e60cbbf2036eb9661
     Subject: Move sources to end of module (a94fe231)
        Date: 2021-03-20 10:23:16 +0000
```

I downloaded the os image from here:
[Operating system images](https://www.raspberrypi.org/software/operating-systems/)

**Comment by ghisvail** [comment]
I was able to reproduce the issue with the steps above. Could you verify whether the same happens with another distribution of the software (official deb or rpm)?

**Comment by ghisvail** [comment]
Just to confirm whether it is specific to the Flatpak version or an issue affecting the software broadly.

**Comment by ghisvail** [comment]
Could you post a screenshot of what the end result looks like?

**Comment by Samet-MohamedAmin** [comment]
I will try it from the official repository and take a screenshot for the desired end result

**Comment by Samet-MohamedAmin** [comment]
On ubuntu 20 using the official `.deb` package.
After I selected the `.img` file, I ended up with this result:

![](https://i.imgur.com/EgzhTZ7.png)


---
system info:

```
$ cat /etc/*release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.2 LTS"
NAME="Ubuntu"
VERSION="20.04.2 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.2 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
```

**Comment by ghisvail** [comment]
Should be fixed by #12. Feel free to reopen if that's not the case.
